recipes/pdspy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+    name: pdspy
+    version: "1.5.0"
+
+source:
+    url: https://files.pythonhosted.org/packages/ee/b0/3e834fb25a597ef12a9764b92ee1c5cf6e1d4897805c9c1346ac0f70dfbf/pdspy-1.5.0.tar.gz
+    sha256: 94d68282132f0272ef41fa4da84eebbf859ead2f3cb407223feeb9f2d1a7a247
+
+build:
+    number: 0
+    script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+
+requirements:
+    host:
+        - python
+        - setuptools
+        - numpy
```

Comment:
if you are building against numpy, please see:

https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
Have updated this, thanks!